lib/actions/api.js

```diff
@@ -530,8 +536,26 @@ export function findStopsWithinBBox (params) {
 
 export const clearStops = createAction('CLEAR_STOPS_OVERLAY')
 
+const throttledUrls = {}
+
+function now () {
+  return (new Date()).getTime()
+}
+
+const TEN_SECONDS = 10000
+
+// automatically clear throttled urls older than 10 seconds
+setInterval(() => {
```

Comment:
I like including `window.` on globals like `setInterval` -- I think it makes it easier to search for these types of functions in the code. Also, I don't think the comment fully describes what the function is doing and this kind of standalone, repeating execution deserves a good description.

Reply:
Will add